erigon-lib/state/aggregator2.go

```diff
@@ -59,20 +59,63 @@ var dbgCommBtIndex = dbg.EnvBool("AGG_COMMITMENT_BT", false)
 
 func init() {
 	if dbgCommBtIndex {
-		cfg := Schema[kv.CommitmentDomain]
-		cfg.AccessorList = AccessorBTree | AccessorExistence
-		Schema[kv.CommitmentDomain] = cfg
+		Schema.CommitmentDomain.AccessorList = AccessorBTree | AccessorExistence
 	}
+	InitSchemas()
+	InitAccountSchemaIntegrity()
```

Comment:
maybe it's better to have all of this in single place? I don't mind the `Schema = SchemaGen{...` and put everything there?